core/spec.md

```diff
@@ -1962,53 +2063,65 @@ Link: <https://example.com/endpoints/123/definitions&page=2>;rel=next;count=100
 }
 ```
 
-##### Creating Resources
+#### Creating Resources
+
+To create a new Resource, there are 5 different HTTP actions that MAY be used:
+- `POST /GROUPs/gID/RESOURCEs`
+- `PUT  /GROUPs/gID/RESOURCEs/rID`
+- `POST /GROUPs/gID/RESOURCEs/rID`
+- `POST /GROUPs/gID/RESOURCEs/rID/versions`
+- `PUT  /GROUPs/gID/RESOURCEs/rID/versions/vID`
 
-To create a new Resource, an HTTP `POST` MAY be used.
+All five follow the the same overall semantics rules so they are described
```

Comment:
semantics rules => semantic rules?

(I tend to use semantics as a noun, but semantic as an adjective... not sure if that's "correct" or just personal habit.)

Reply:
(This occurs quite a few times.)

Reply:
got 'em!